blazar/plugins/oshosts/host_plugin.py

```diff
@@ -618,6 +646,6 @@ def _check_params(self, values):
             raise manager_ex.MissingParameter(param='resource_properties')
 
         if 'before_end' not in values:
-            values['before_end'] = 'default'
+            values['before_end'] = 'email'
```

Comment:
The `default` setting looked to have special meaning before: https://github.com/ChameleonCloud/blazar/pull/2/files#diff-91ae8d7b3d68e5dd64afbbeb810e4995R346 - it would instruct Blazar to instead look at the configuration to see what action should be taken, based on the resource type associated with the lease. As such I don't think it should be overridden. If we want to email by default that should be placed in the Blazar configuration (which should work out of the box, nicely!)